components/actionbutton/index.css

```diff
@@ -68,8 +78,51 @@
+		--mod-actionbutton-content-color-focus: var(--mod-actionbutton-static-black-content-color-focus, var(--spectrum-black));
+
+		--mod-actionbutton-content-color-default-selected: var(--mod-actionbutton-static-black-content-color-selected, var(--spectrum-white));
+		--mod-actionbutton-content-color-hover-selected: var(--mod-actionbutton-static-black-content-color-hover-selected, var(--spectrum-white));
+		--mod-actionbutton-content-color-down-selected: var(--mod-actionbutton-static-black-content-color-down-selected, var(--spectrum-white));
+		--mod-actionbutton-content-color-focus-selected: var(--mod-actionbutton-static-black-content-color-focus-selected, var(--spectrum-white));
+
+		--spectrum-actionbutton-content-color-disabled: var(--spectrum-disabled-static-black-content-color);
+		--spectrum-actionbutton-focus-indicator-color: var(--spectrum-static-black-focus-indicator-color);
+	}
+
+	&.spectrum-ActionButton--staticWhite {
+		/* background color _not_ selected is coming from the system theme layer */
+		--spectrum-actionbutton-background-color-default-selected: var(--spectrum-transparent-white-800);
+		--spectrum-actionbutton-background-color-hover-selected: var(--spectrum-transparent-white-900);
+		--spectrum-actionbutton-background-color-down-selected: var(--spectrum-transparent-white-900);
+		--spectrum-actionbutton-background-color-focus-selected: var(--spectrum-transparent-white-900);
+
+		/* note the introduction of a more specific modifier in the following stacks, those should be used instead for customizations */
+		--mod-actionbutton-content-color-default: var(--mod-actionbutton-static-white-content-color, var(--spectrum-white));
+		--mod-actionbutton-content-color-hover: var(--mod-actionbutton-static-white-content-color-hover, var(--spectrum-white));
+		--mod-actionbutton-content-color-down: var(--mod-actionbutton-static-white-content-color-down, var(--spectrum-white));
+		--mod-actionbutton-content-color-focus: var(--mod-actionbutton-static-white-content-color-focus, var(--spectrum-white));
+
+		--mod-actionbutton-content-color-default-selected: var(--mod-actionbutton-static-white-content-color-selected, var(--spectrum-black));
+		--mod-actionbutton-content-color-hover-selected: var(--mod-actionbutton-static-white-content-color-selected, var(--spectrum-black));
+		--mod-actionbutton-content-color-down-selected: var(--mod-actionbutton-static-white-content-color-selected, var(--spectrum-black));
+		--mod-actionbutton-content-color-focus-selected: var(--mod-actionbutton-static-white-content-color-selected, var(--spectrum-black));
+
+		--spectrum-actionbutton-content-color-disabled: var(--spectrum-disabled-static-white-content-color);
+		--spectrum-actionbutton-focus-indicator-color: var(--spectrum-static-white-focus-indicator-color);
+	}
+
+	/* Emphasized styles are only applied to the "selected" state of the button and are not applied in static color modes; must list after static variants */
+	&.spectrum-ActionButton--emphasized {
```

Comment:
I think the simplest or most straightforward way to get the static color + emphasized background colors corrected is to add the `:not(.spectrum-ActionButton--staticBlack,.spectrum-ActionButton--staticWhite)` back to this selector. Do you have reservations about doing that `:not()`?

Reply:
ooorrrr....we adjust the template. Does that work? That would reduce the need to continue to adjust the CSS you have right now.

`[`${rootClass}--emphasized`]: isEmphasized && !staticColor,`, that way the --emphasized class is only applied when no static color arg has been defined.

Reply:
Oh good point, I wonder if the web component can enforce that too. 🤔

Reply:
I added back the not selectors since SWC already accounts for it!